Dockerfile-test

```diff
@@ -0,0 +1,17 @@
+ARG UBUNTU_VER=bionic-20190912.1
```

Comment:
What's this Dockerfile for?

Reply:
OSX didn't seem to have an easy way to get the libsystemd header files to appropriately run the unit tests -- so I just made a quicky Dockerfile to have a test environment in Ubuntu with the same libs that the production images would have.